custom_components/smartthinq_sensors/wideq/devices/washerDryer.py

```diff
@@ -124,6 +126,13 @@ def __init__(
         self._course_keys: dict[CourseType, str | None] | None = None
         self._course_infos: dict[str, str] | None = None
         self._selected_course: str | None = None
+        self._course_overrides: dict | None = {}
+        self._course_overrides_lists: dict | None = {
+            'temp': ['TEMP_COLD', 'TEMP_20', 'TEMP_30', 'TEMP_40', 'TEMP_60', 'TEMP_95'],
+            'spin': ['NO_SPIN', 'SPIN_400', 'SPIN_800', 'SPIN_1000', 'SPIN_Max'],
+            'rinse':  ['RINSE_NORMAL', 'RINSE_PLUS']}
+            # Need to initialise this list so that the UI can setup the select drop down.
+            # A better solution would be to generate this information from the data returned ThinQ API.
```

Comment:
This values can be different depending on device model and should be retrieved from model info. This could work for your device but not for other. I need to be sure that only supported modes can be used.

Reply:
@ollo69 

> This values can be different depending on device model and should be retrieved from model info. 

Now retrieved from model info.